docker_test.go

```diff
@@ -15,6 +15,74 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
+func TestContainerAttachedToNewNetwork(t *testing.T) {
+	networkName := "new-network"
```

Comment:
Hello,
I tried your branch and one of the network created in this test didn't got removed:

```
 $ docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
6050f659c458        bridge              bridge              local
adbb6c9dfe3f        host                host                local
c40a6b07ec52        new-network         bridge              local
dccf217218e7        none                null                local
ff00e6906c6e        potato_net-test     bridge              local
```
Do you have an idea?

Reply:
Shouldn't this line remove it? https://github.com/testcontainers/testcontainers-go/pull/76/files/22d6b6a6e5d8bc2a0fcad730929783ad88a3c5d2#diff-98b81bf788c107c9f5f395060629d5ecR48

Reply:
Fixed! I sent this before your fix edb1f310022e48268867572431fb2a467c85c0b5, so I was following same pattern. I pushed-force one commit after rebasing with master

Thank you!